solutions/array/frequency-101.js

```diff
@@ -0,0 +1,24 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arr=readLine().split(' ');
+let n=parseInt(arr[0]);
+let num=parseInt(arr[1]);
```

Comment:
@md-qubais  please use the names mentioned in the problem description    `N` and `k` in place of n,num

Reply:
okay